buildbot/master.cfg

```diff
@@ -340,10 +341,16 @@ def make_dolphin_osx_build(mode="normal"):
                            descriptionDone="sign",
                            haltOnFailure=True))
 
+    f.addStep(ShellCommand(command="/build/codesign.sh --deep Binaries/Dolphin\ Updater.app",
+                           workdir="build/build",
+                           description="signing updater",
+                           descriptionDone="sign updater",
+                           haltOnFailure=True))
 
     f.addStep(ShellCommand(command=["hdiutil", "create", "dolphin.dmg",
                                     "-format", "UDBZ",
-                                    "-srcfolder", "Binaries/dolphin.app", "-ov",
+                                    "-layout", "SPUD", # Needed for 7-Zip support
+                                    "-srcfolder", "Binaries/Dolphin.app", "-ov",
```

Comment:
does case of "dolphin" matter here?

Reply:
I do think so. It's probably the cause for ``dolphin.app`` not being capitalized as ``Dolphin.app`` as it should.

Reply:
I meant: "-srcfolder Binaries/Dolphin.app" implies "Binaries/Dolphin.app" already exists. Is that the case? Or this change runs risk of breaking things.

Reply:
Used before with codesign.sh. pretty sure it won't cause issues

Reply:
Might be caused by us using APFS which is not case sensitive...